_opensearch/logs.md

```diff
@@ -168,7 +168,36 @@ node1 | [2019-10-24T19:48:51,012][WARN][i.i.s.index] [node1] [some-index/i86iF5k
 
 Slow logs can consume considerable disk space if you set thresholds or levels too low. Consider enabling them temporarily for troubleshooting or performance tuning. To disable slow logs, return all thresholds to `-1`.
 
+## Task logs
+
+OpenSearch can log CPU time and memory utilization for the top N expensive search tasks when task resource consumers are enabled. By default, task resource consumers will log the top 10 search tasks at 60 second intervals. These values can be configured in `opensearch.yml`.
```

Comment:
Can we rephrase to ```top N memory expensive search tasks``` to make it more accurate?

Reply:
Will do!